src/makeCalculator.js

```diff
@@ -37,7 +37,26 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    add: (a, b) => a + b,
+    subtract: (a, b) => a - b,
+    divide: (a, b) => a / b,
+    multiply: (a, b) => a * b,
+    result: 0,
+
+    operate(operation, digit) {
```

Comment:
`digit` is incorrect variable name, because you can operate on a number that consists of multiple digits, it would be better to call it `number` or `value`

Reply:
done :P